app/services/show_all_seats.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class ShowAllSeats
```

Comment:
this is a small case that we prefer to call services/usecases that *do* something using verbs but also tables/models/representers using nouns. In this case it could mean without looking into a code that it's a service *showing all seats* like all seats not just *show's seats* :D